gnuradio-runtime/lib/constants.cc.in

```diff
@@ -1,6 +1,7 @@
 /* -*- c++ -*- */
 /*
  * Copyright 2006,2009,2013 Free Software Foundation, Inc.
+ * Copyright 2021 GNU Radio contributors
```

Comment:
This line is pointless. In this case, I would suggest just leaving it out. It's not false, but it doesn't actually tell a story that's not covered better by git. And this particular contribution to this file is also not really that amazing that it warrants a new copyright owner.

Reply:
true, was more of a mental automatism.